.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -4,24 +4,57 @@ about: Request our thoughts on your suggestion for a new feature for Homebrew.
 
 ---
 
-# Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.
+<details>
```

Comment:
Just noticed I didn’t submit this comment (damn ⌘↵ starting a review).

Hiding the big message on `details` is pretty smart! I think I might steal that idea for the cask bug report template.